bee-queue/arena
# Feat flows view

#### Changes Made
This PR adds new views to support flows creation (this is only available for Bullmq for now), so when the other queues allows the something similar, we could use the same pattern
#### Potential Risks
We would have the ability to create parent-children jobs in a single page view
#### Test Plan
1. Go to example directory
2. Run npm run start:bullmq
3. Go to Flows Overview
4. Enter to the first flow in the list
5. Click in the Add Flow button
6. Copy and paste this object:
```js
{
    "name": "parent-job2",
    "queueName": "name_of_my_parent_queue",
    "data": {},
    "children": [
      {"name": "child", "data": {"idx": 0, "foo": "bar"}, "queueName": "name_of_my_queue"},
      {"name": "child", "data": {"idx": 2, "foo": "bar"}, "queueName": "name_of_my_queue"},
{"name": "child", "data": {"idx": 3, "foo": "bar"}, "queueName": "name_of_my_queue"}
    ]
  }
```
7. All the jobs should be created, you now can go to queues overview and look for them, the parent should be in waiting-children state

Notes: 
1. Also I'm checking if the flows options is available, so bull and bee queues should not have the ability to see these views since this feature is not available for them yet.
2. As flows has a connection as the queues, it is following the same structure
3. Documentation for flows https://docs.bullmq.io/guide/jobs/flows

#### Checklist
